Move configuration files to a more conventional location.

From: Roman Donchenko <roman.donchenko@itseez.com>

This patch is originally from <https://kforge.ros.org/openni/drivers/>.
---
 Source/OpenNI/XnOpenNI.cpp |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Source/OpenNI/XnOpenNI.cpp b/Source/OpenNI/XnOpenNI.cpp
index 92063cf..de4d7e7 100644
--- a/Source/OpenNI/XnOpenNI.cpp
+++ b/Source/OpenNI/XnOpenNI.cpp
@@ -6991,9 +6991,9 @@ XN_C_API XnStatus xnScriptNodeRun(XnNodeHandle hScript, XnEnumerationErrors* pEr
 #if (XN_PLATFORM == XN_PLATFORM_WIN32)
 	#define XN_OPEN_NI_FILES_LOCATION "\\Data\\"
 #elif (CE4100)
-	#define XN_OPEN_NI_FILES_LOCATION "/usr/etc/ni/"
+	#define XN_OPEN_NI_FILES_LOCATION "/etc/openni/"
 #elif (XN_PLATFORM == XN_PLATFORM_LINUX_X86 || XN_PLATFORM == XN_PLATFORM_LINUX_ARM || XN_PLATFORM == XN_PLATFORM_MACOSX)
-	#define XN_OPEN_NI_FILES_LOCATION "/var/lib/ni/"
+	#define XN_OPEN_NI_FILES_LOCATION "/etc/openni/"
 #elif (XN_PLATFORM == XN_PLATFORM_ANDROID_ARM)
 	#define XN_OPEN_NI_FILES_LOCATION "/data/ni/"
 #else
